modules/preview-react/table/lib/TableDivider.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {BoxProps, Box} from '@workday/canvas-kit-react/layout';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {createComponent} from '@workday/canvas-kit-react/common';
+
+export const TableDivider = createComponent('hr')({
+  displayName: 'Table.Divider',
+  Component({...elemProps}: BoxProps, ref, Element) {
+    return (
+      <Box
+        as={Element}
+        ref={ref}
+        border={0}
```

Comment:
Does the box by default add a border? If you just have borderBottom do you still need to set this to 0?

Reply:
Yeah it does. So for the `hr` element, it will add a border all the way around the element which makes it look like a box